src/ui/controls/shortcut/index.tsx

```diff
@@ -88,11 +89,16 @@ export default function ShortcutLink(props: ShortcutLinkProps) {
         });
     }
 
+    function onUpdate(props) {
```

Comment:
Let's rename this function to `on Render` (see why later). `props` is an HTMLElement (HTMLAElement to be more precise), we can rename the argument and add a type to it (since we are using TypeScript).